Android.mk

```diff
@@ -69,6 +69,13 @@ LOCAL_C_INCLUDES += $(LOCAL_PATH)/host/xtest \
 		$(LOCAL_PATH)/ta/aes_perf/include \
 		$(LOCAL_PATH)/ta/socket/include
 
+ifeq ($(CFG_SECURE_DATA_PATH),y)
+CFLAGS += -DWITH_SECURE_DATA_PATH
```

Comment:
Use `CFG_SECURE_DATA_PATH` instead of `WITH_SECURE_DATA_PATH`, then you don't need to set `CFLAGS`.